src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1400,6 +1405,11 @@ private void EnterParameters(ImmutableArray<ParameterSymbol> parameters)
 
         protected virtual void EnterParameter(ParameterSymbol parameter)
         {
+            if ((parameter.ContainingSymbol as MethodSymbol)?.MethodKind == MethodKind.LocalFunction)
+            {
+                _variableUsePass.Visit(parameter.ExplicitDefaultExpression);
```

Comment:
No. The only thing that `_variableUsePass` mutates is `_usedVariables` and `_usedLocalFunctions`, both of which are only read when reporting `WRN_UnreferencedVar` and friends.